### PR TITLE
Modify SpriteLabs buttons to remove hover action when disabled

### PR DESCRIPTION
**Pull request by ebeastlake**
Currently in Sprite Labs, when the arrow buttons or pause button are disabled, they still look clickable because the cursor changes to the "pointer" on hover.

Setting the `pointer-events` are set to `none` for arrow and pause buttons with the disabled attribute fixes this issue.

See video below:

https://user-images.githubusercontent.com/26844240/197296176-50095589-280c-4d8f-bda7-ebe6e7889f54.mov

## Links

Jira ticket: [SL-283](https://codedotorg.atlassian.net/browse/SL-283)

## Testing story

Tested manually.

https://user-images.githubusercontent.com/26844240/197296943-64ec0357-55f5-428f-82ac-48a779b91d80.mov

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by ebeastlake** [comment]
I couldn't make similar arrow buttons appear in GameLab even though they're in shared code, but will be looking for confirmation that this isn't a change we should be applying/testing there too. Also, it looks like there are some debugger buttons in GameLab with a similar issue. Could fix them as part of this PR (and possibly go hunting for other cases?) or file a follow-up.

**Review by molly-moen — APPROVED** [review]
LGTM! If you find buttons with a similar issue I would file a follow-up ticket

File: apps/style/spritelab/style.scss [hunk 1]
```diff
@@ -1 +1,9 @@
 @import "../gamelab/style.scss";
+
+button.arrow:disabled {
+  pointer-events: none;
+}
+
+button#pauseButton:disabled {
+  pointer-events: none;
+}
```
Comment (maddiedierker) [inline apps/style/spritelab/style.scss]: i don't think we should do this here, but if it's useful in the future, Sass has an [`@extend` rule](https://sass-lang.com/documentation/at-rules/extend)

Comment (maddiedierker) [reply apps/style/spritelab/style.scss]: we could also do something like:

```scss
button {
  &.arrow:disabled,
  &#pauseButton:disabled {
    pointer-events: none;
  }
}
```

**Review by maddiedierker — APPROVED** [review]

**Pull request merged**